client/src/components/leftPanel/EditPortfolios.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react'
+import { withRouter } from 'react-router-dom'
+
+import { createPortfolio, deletePortfolio } from '../../utils/serverApi'
+import deleteIcon from '../../assets/delete.svg'
+import './EditPortfolios.scss'
+import './TagEdit.scss'
+import Modal from './Modal'
+import Loader from '../ui/Loader'
+
+class EditPortfolios extends Component {
+  state = {
+    addInput: '',
+    deleteSearchInput: '',
+    loading: false,
+    resultType: ''
+  }
+
+  onClose = () => {
+    this.props.history.push(`/`)
```

Comment:
Maybe you want to do the same check as in `TagEdit`. Or maybe not 🤔 
```javascript
if (this.props.history.length > 2) {
  this.props.history.goBack()
} else {
  // in case tag edit was first visited site
  this.props.history.push(`/${encodeURI(this.props.tag.tag)}`)
}
```

Reply:
I was thinking that this will not be used too often to need it behave in more intelligent way, and if you delete some portfolios it was easier just to reset the page, rather than checking portfolios in the url filter parameters.